datafusion/physical-expr-common/src/binary_view_map.rs

```diff
@@ -149,7 +149,7 @@ where
             output_type,
             map: hashbrown::raw::RawTable::with_capacity(INITIAL_MAP_CAPACITY),
             map_size: 0,
-            builder: GenericByteViewBuilder::new().with_block_size(2 * 1024 * 1024),
+            builder: GenericByteViewBuilder::new(),
```

Comment:
@XiangpengHao  I would appreciate your help with this change

In https://github.com/apache/datafusion/pull/11862 you set this to `with_fixed_block_size`: 
https://github.com/apache/datafusion/pull/11862/files#diff-0007996e12eb1b2e63363974424f13330f85a7fc48e0c55381f8a30a0f372931R405

However, since then you have added the dynamic growth size in https://github.com/apache/arrow-rs/pull/6136  which seems like it would be better

Would it be possible for you to run your benchmarks again to make sure this doesn't cause regressions?

Reply:
Running 🏃 , will report back in a few minutes

Reply:
I think there's no regression...

```
┏━━━━━━━━━━━━━━┳━━━━━━━━━━━┳━━━━━━━━━━━┳━━━━━━━━━━━━━━━┓
┃ Query        ┃   old     ┃    new    ┃        Change ┃
┡━━━━━━━━━━━━━━╇━━━━━━━━━━━╇━━━━━━━━━━━╇━━━━━━━━━━━━━━━┩
│ QQuery 0     │    0.32ms │    0.30ms │ +1.07x faster │
│ QQuery 1     │   42.22ms │   42.70ms │     no change │
│ QQuery 2     │   77.51ms │   75.82ms │     no change │
│ QQuery 3     │   87.90ms │   83.73ms │     no change │
│ QQuery 4     │  768.94ms │  766.91ms │     no change │
│ QQuery 5     │  894.50ms │  897.62ms │     no change │
│ QQuery 6     │   40.50ms │   41.16ms │     no change │
│ QQuery 7     │   43.33ms │   43.97ms │     no change │
│ QQuery 8     │ 1204.09ms │ 1189.47ms │     no change │
│ QQuery 9     │  888.97ms │  888.96ms │     no change │
│ QQuery 10    │  218.92ms │  217.11ms │     no change │
│ QQuery 11    │  241.49ms │  240.46ms │     no change │
│ QQuery 12    │  939.97ms │  932.35ms │     no change │
│ QQuery 13    │ 1494.84ms │ 1518.03ms │     no change │
│ QQuery 14    │ 1370.17ms │ 1351.12ms │     no change │
│ QQuery 15    │  877.87ms │  889.39ms │     no change │
│ QQuery 16    │ 2602.40ms │ 2607.00ms │     no change │
│ QQuery 17    │ 2508.71ms │ 2516.16ms │     no change │
│ QQuery 18    │ 5436.93ms │ 5531.71ms │     no change │
│ QQuery 19    │   77.64ms │   76.50ms │     no change │
│ QQuery 20    │  831.26ms │  828.42ms │     no change │
│ QQuery 21    │ 1037.75ms │ 1026.61ms │     no change │
│ QQuery 22    │ 2799.46ms │ 2771.03ms │     no change │
│ QQuery 23    │ 9135.71ms │ 9116.49ms │     no change │
│ QQuery 24    │  484.32ms │  484.82ms │     no change │
│ QQuery 25    │  440.32ms │  433.59ms │     no change │
│ QQuery 26    │  546.80ms │  544.41ms │     no change │
│ QQuery 27    │ 1230.39ms │ 1237.77ms │     no change │
│ QQuery 28    │ 8683.69ms │ 8632.35ms │     no change │
│ QQuery 29    │  361.55ms │  356.75ms │     no change │
│ QQuery 30    │ 1047.39ms │ 1052.35ms │     no change │
│ QQuery 31    │ 1464.36ms │ 1462.84ms │     no change │
│ QQuery 32    │ 7442.01ms │ 7410.51ms │     no change │
│ QQuery 33    │ 3340.53ms │ 3356.63ms │     no change │
│ QQuery 34    │ 3345.35ms │ 3358.90ms │     no change │
│ QQuery 35    │ 1484.52ms │ 1477.14ms │     no change │
│ QQuery 36    │  151.45ms │  150.11ms │     no change │
│ QQuery 37    │   93.40ms │   97.17ms │     no change │
│ QQuery 38    │   95.15ms │  101.52ms │  1.07x slower │
│ QQuery 39    │  447.55ms │  456.85ms │     no change │
│ QQuery 40    │   50.42ms │   49.59ms │     no change │
│ QQuery 41    │   46.07ms │   44.46ms │     no change │
│ QQuery 42    │   52.52ms │   52.71ms │     no change │
└──────────────┴───────────┴───────────┴───────────────┘
┏━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━┓
┃ Benchmark Summary      ┃            ┃
┡━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━┩
│ Total Time (results)   │ 64429.19ms │
│ Total Time (results)   │ 64413.50ms │
│ Average Time (results) │  1498.35ms │
│ Average Time (results) │  1497.99ms │
│ Queries Faster         │          1 │
│ Queries Slower         │          1 │
│ Queries with No Change │         41 │
└────────────────────────┴────────────┘
```

Reply:
Awesome -- thank you for checking.